modules/brew/brew.sh

```diff
@@ -277,4 +282,64 @@ if [[ "${BREW_ANALYTICS}" == false ]]; then
   fi
 fi
 
+# Create directory for brew configuration
+mkdir -p /usr/share/bluebuild/brew
+
+# Create repo-info.yml file with install packages if specified
+if [[ -n "${INSTALL_PACKAGES}" ]]; then
+    echo "install:" > /usr/share/bluebuild/brew/repo-info.yml
+    echo "${INSTALL_PACKAGES}" | sed 's/^/  - /' >> /usr/share/bluebuild/brew/repo-info.yml
+    echo "The following Brew packages will be installed when the system is live:"
+    echo "${INSTALL_PACKAGES}" | sed 's/^/  - /'
+
+    # Write brew-packages-setup script
+    cat > /usr/bin/brew-packages-setup <<EOF
```

Comment:
This should be just a file in the module's directory that is `cp`'d in place. Having it inline doesn't work with syntax highlighting and shellcheck.